octavia_f5/controller/worker/l2_sync_manager.py

```diff
@@ -195,6 +195,10 @@ def ensure_l2_flow(self, selfips: [network_models.Port], network_id: str, device
             if device and bigip.hostname != device:
                 continue
 
+            # Check if device available by symple GET request
+            if not bigip.is_available(timeout=CONF.status_manager.failover_timeout):
```

Comment:
Please log a warning if the BigIP isn't available.

Reply:
Fixed in the last commit.